packages/medusa/src/api/routes/store/order-edits/get-order-edit.ts

```diff
@@ -0,0 +1,59 @@
+import { Request, Response } from "express"
+import { OrderEditService } from "../../../../services"
+
+/**
+ * @oas [get] /order-edits/{id}
+ * operationId: "GetOrderEditsOrderEdit"
+ * summary: "Retrieve an OrderEdit"
+ * description: "Retrieves a OrderEdit."
+ * parameters:
+ *   - (path) id=* {string} The ID of the OrderEdit.
+ * x-codeSamples:
+ *   - lang: JavaScript
+ *     label: JS Client
+ *     source: |
+ *       import Medusa from "@medusajs/medusa-js"
+ *       const medusa = new Medusa({ baseUrl: MEDUSA_BACKEND_URL, maxRetries: 3 })
+ *       medusa.orderEdit.retrieve(orderEditId)
+ *       .then(({ order_edit }) => {
+ *         console.log(order_edit.id);
+ *       });
+ *   - lang: Shell
+ *     label: cURL
+ *     source: |
+ *       curl --location --request GET 'https://medusa-url.com/store/order-edits/{id}'
+ * tags:
+ *   - OrderEdit
+ * responses:
+ *   200:
+ *     description: OK
+ *     content:
+ *       application/json:
+ *         schema:
+ *           properties:
+ *             order_edit:
+ *               $ref: "#/components/schemas/order_edit"
+ *   "400":
+ *     $ref: "#/components/responses/400_error"
+ *   "401":
+ *     $ref: "#/components/responses/unauthorized"
+ *   "404":
+ *     $ref: "#/components/responses/not_found_error"
+ *   "409":
+ *     $ref: "#/components/responses/invalid_state_error"
+ *   "422":
+ *     $ref: "#/components/responses/invalid_request_error"
+ *   "500":
+ *     $ref: "#/components/responses/500_error"
+ */
+export default async (req: Request, res: Response) => {
+  const orderEditService: OrderEditService =
+    req.scope.resolve("orderEditService")
+
+  const { id } = req.params
+  const retrieveConfig = req.retrieveConfig
+
+  const orderEdit = await orderEditService.retrieve(id, retrieveConfig)
+
+  return res.json({ order_edit: orderEdit })
```

Comment:
todo: this response should include the computed line items; think it would make sense to have something similar to a: `orderEditService.computeLineItems(orderEdit)` method, that we could call here and then set `orderEdit.items` before responding.

Reply:
Same goes for the `/admin` response

Reply:
yes you are right, should I include a separate object on the order edit which is the `removed_items` as we have populated in the code snippet in notion?

Reply:
I added it, but before the next review I need to add the integration tests related to it. I ll go to eat and then do it